lib/flow_runner/custom_blocks/whatsapp_template_message.ex

```diff
@@ -18,25 +18,53 @@ defmodule FlowRunner.CustomBlocks.WhatsAppTemplateMessage do
       template: %{
         name: name,
         language: %{code: code},
-        components: Enum.map(components, &parse_component/1),
+        components: Enum.map(components, &parse_component(&1, code)),
         tracking: Map.get(template, "tracking")
       }
     }
   end
 
-  def parse_component(%{"type" => type, "parameters" => parameters} = component),
+  @spec parse_component(
+          %{
+            required(String.t()) => String.t(),
+            required(String.t()) => list(%{String.t() => String.t()})
+          },
+          String.t()
+        ) :: %{
+          type: String.t(),
+          index: String.t() | nil,
+          sub_type: String.t() | nil,
+          parameters:
+            list(%{
+              required(:type) => String.t(),
+              optional(:text) => String.t(),
+              optional(:payload) => String.t(),
+              optional(:language) => String.t()
+            })
+        }
+  defp parse_component(
+         %{"type" => type, "parameters" => parameters} = component,
+         template_language
+       ),
+       do: %{
+         type: type,
+         index: component["index"],
+         # required only for buttons
+         sub_type: component["sub_type"],
+         parameters: Enum.map(parameters, &parse_parameter(&1, template_language))
+       }
+
+  defp parse_parameter(%{"type" => "text", "text" => text} = component, template_language),
     do: %{
-      type: type,
-      index: component["index"],
-      # required only for buttons
-      sub_type: component["sub_type"],
-      parameters: Enum.map(parameters, &parse_parameter/1)
+      type: "text",
+      text: text,
+      language: component["language"] || Expression.evaluate_block!(template_language)
```

Comment:
This is the main change of this PR

Reply:
@joaothallis Would it be the same if we just do this?
```suggestion
      language: component["language"] || template_language
```

Reply:
No, `Expression.evaluate_block!(template_language)` returns: `"en"`. `template_language` returns: `"\"en\""`, it breaks the expected format